src/modules/tracker.js

```diff
@@ -218,26 +218,26 @@ class Tracker {
         name,
         item_id,
         customer_id,
+        customerId = customer_id,
```

Comment:
Do we need the customerId here?

Doesn't seem like we had it before so no one uses it probably?